activiti-cloud-modeling-service/activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/java/org/activiti/cloud/services/modeling/validation/extensions/TaskMappingsServiceTaskImplementationValidator.java

```diff
@@ -173,4 +181,9 @@ private String getImplementationKey(String connectorName,
                             connectorName,
                             action.getName());
     }
+
+    private void addDmnScriptImplementations(Map<String, ConnectorModelFeature> availableConnectorActions){
```

Comment:
this is private and unused right?

Reply:
Yes I must delete it. It is from the first approach that I discarded finally.